node/resolvers/appsWithStats.ts

```diff
@@ -1,14 +1,27 @@
-import { removeVersionFromAppId } from '@vtex/api'
-import { compose, filter, isEmpty, isNil, keys, map, reject, startsWith } from 'ramda'
+import { filter, includes, map, pluck, prop } from 'ramda'
 
-const nullable = x => isEmpty(x) || isNil(x)
+
+const filterAppNameByAccount = (appNameList: string[], accountName: string) => {
+  const isVendorInApp = (appName: string) => {
+    const appNamePart: string[] = appName.split('.')
+    return includes(accountName, appNamePart[0])
```

Comment:
I think `includes` is not appropriate here. 
It is unlikely, but consider this: `vendorvtexvendor.appName` can be interpreted as an app from vendor `vtex`.